src/test/java/Homework17.java

```diff
@@ -0,0 +1,46 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class Homework17 {
+
+    @Test(enabled = true, priority = 0, description = "addSongToPlaylist")
+    public static void addSongToPlaylist() throws InterruptedException {
+
+        WebDriver driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+
+        String url = "https://bbb.testpro.io/";
+        driver.get(url);
+        Assert.assertEquals(driver.getCurrentUrl(), url);
+
+        WebElement emailField = driver.findElement(By.cssSelector("[type='Email']"));
+        emailField.sendKeys("andrey.27.avd@gmail.com");
+
+        WebElement passwordField = driver.findElement(By.cssSelector("[type='password']"));
+        passwordField.sendKeys("te$t$tudent");
+
+        WebElement button = driver.findElement(By.cssSelector("[type='submit']"));
+        button.click();
+
+        Thread.sleep(2000);
+
+        WebElement songs = driver.findElement(By.cssSelector("[class='songs']"));
+        songs.click();
+
+        WebElement likeButton = driver.findElement(By.cssSelector("[title='Like Episode 2 by Music Insiders By Fma'][data-test='like-btn']"));
```

Comment:
You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/209942669-5c315c74-e8a7-4811-892c-603463a26d4b.png)

For example xpath  `//td[contains( text(),'Episode 2')]`